contracts/multisig/src/error.rs

```diff
@@ -22,6 +22,9 @@ pub enum ContractError {
     #[error("{signer:?} submitted an invalid signature for signing session {session_id:?}")]
     InvalidSignature { session_id: Uint64, signer: String },
 
+    #[error("signed sender address could not be verified using submitted public key")]
+    InvalidPublicKeyRegistrationSignature,
```

Comment:
Do we need a new error type here? We can probably just reuse `InvalidSignature` above.

Reply:
that was my original thought, but the error above requires session_id which is not available for public key registration